src/sentry/api/endpoints/group_details.py

```diff
@@ -179,14 +179,6 @@ def get(self, request, group):
         :auth: required
         """
         # TODO(dcramer): handle unauthenticated/public response
```

Comment:
Can this todo get removed?

Reply:
probably 🤷‍♀️ not entirely sure what it means